tests/automation_framework/src/libs/test_base.py

```diff
@@ -14,14 +14,12 @@ def __init__(self):
         self.uri_client = env.uri_client
         self.build_request_output = {}
 
-    def setup_and_build_request_register(self, input_file):
-        pre_test_output = pre_test_worker_env(input_file)
```

Comment:
Removed the pre_test_worker_env call as there is nothing to be done for worker_register.